addons/sale/controllers/portal.py

```diff
@@ -123,7 +123,8 @@ def portal_order_page(
         access_token=None,
         message=False,
         download=False,
-        downpayment=None,
+        installment=None,
+        link_amount=None,
```

Comment:
"link amount" in a link sounds weird... `downpayment_amount` would make more sense, in particular if it's only passed when < 100% of the full amount.

Reply:
I am not sure if downpayment_amount name is also fitting, as if it's already sale order, then it wouldn't be downpayment anymore, but the link_amount will still be sent to enable paying, and when it is sale order, every link amount including one equal to amount_total must be sent.

Reply:
Actually, link_amount must be passed in all cases because when it is the link amount, even when the Sale Order requires a signature, the payment link allows the payment to be made without a signature(on purpose).

Reply:
I already used a name that would fit the name of the variable, installment. And that's what it also is when quotation is already sale order but is not paid yet. Even when installment_amount = amount_total it still makes sense, as it might be treated as 1 installment kind of payment. What do you think?